core/src/config/refine_params.rs

```diff
@@ -14,7 +14,7 @@ pub struct RefineParamsData<T: Config> {
 }
 
 impl<T: Config> RefineParamsData<T> {
-    pub(crate) fn new(account_nonce: u64, block_number: u64, block_hash: T::Hash) -> Self {
+    pub fn new(account_nonce: u64, block_number: u64, block_hash: T::Hash) -> Self {
```

Comment:
I guess this needs to be called in Subxt now, so can't be `(crate)`. 

Perhaps we should add `#[doc(hidden)]` instead and a comment that it is not a part of the public API? The reason for this would be so that we can change it without breaking anything. A reason against is that maybe users want to test custom signed exts by using this sort of thing though, so maybe just leaving it pub is ok :)